README.md

```
@@ -1 +1,8 @@
-# xi-web-api
+# Local deployemnt
+
+You can deploy xiVIEW locally using the following three projects:
+- https://github.com/Rappsilber-Laboratory/mzidentml-reader - to create the database xiVIEW accesses and populate this DB with data from mzIdentML files. A local installation the reads CSV files is currently a problem. Follow the instructions in that project.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)